index.html

```diff
@@ -42,10 +42,195 @@ <h2 class="primary-text pt_2">I’m a software developer</h2>
+                    </section>
+                </section>
+            </section>
+        </section>
+
+        <!--About Me-->
+        <section id="about-me">
+            <section class="about-text">
+                <h3 class="about-heading">About me</h3>
+                <img id="about-icon" src="assets/Mask_Group.svg" alt="Background design for about section">
+                <p class="about-desc"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <img id="about-bg" src="assets/aboutFrame.svg" alt="">
+                <button class="about-btn" type="button">Get My Resume</button>
+            </section>
+
+            <ul class="categories">
+                <li class="list-item">
+                    <img src="assets/icon-languages.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Languages</h3>
+
+                        <ul class="languages">
+                            <li class="language">
+                                <span>Javascript</span>
+                            </li>
+                            <li class="language">
+                                <span>Html</span>
+                            </li>
+                            <li class="language">
+                                <span>CSS</span>
+                            </li>
+                        </ul>
+                    </section>
+                </li>
+
+                <li class="list-item">
+                    <img src="assets/icon-frameworks.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Frameworks</h3>
+
+                        <ul class="languages">
+                            <li class="language">
+                                <span>Bootstrap</span>
+                            </li>
+                            <li class="language">
+                                <span>Ruby on rails</span>
+                            </li>
+                        </ul>
+                    </section>
+                </li>
+
+                <li class="list-item">
+                    <img src="assets/icon-skills.svg" alt="Languages Icon">
+
+                    <section class="list-body">
+                        <h3 class="list-title">Skills</h3>
+
+                        <ul class="languages">
+                            <li class="language">
+                                <span>GitHub</span>
+                            </li>
+                            <li class="language">
+                                <span>Terminal</span>
+                            </li>
+                        </ul>
+                    </section>
+                </li>
+            </ul>
```

Comment:
- [ ] Kindly make sure each title such as(**Languages, Skills and Frameworks**) is capitalized like in the Figma design and not in **Uppercase**.

- [ ] Kindy make sure each category has all the items in each card(e.g.: For Frameworks: Ruby on Rails, Bootstrap, Rspec, Selenium,...). Make sure you apply this approach to each card category.